app/src/main/res/values-fr/strings.xml

```diff
@@ -18,11 +18,15 @@
     <string name="acknowledgements">Mentions légales</string>
     <!-- Popup menu item that links to the About page -->
     <string name="about">À propos</string>
-
-    <!-- About page -->
     <!-- External Links label -->
     <string name="external_links">Liens externes</string>
 
+    <!-- External Links labels -->
+    <!-- Icon Request form link -->
+    <string name="request_form">Formulaire de demande d\'icône</string>
```

Comment:
Out of curiosity, is the `\` supposed to be in it or is it a misstype?

Reply:
I'm pretty sure we need to escape single quotes. I did the same things for other strings before:

```xml
    <!-- Subtitle displayed below the user who inspired the app icon -->
    <string name="special_thanks_icon">Pour avoir inspiré l\'icône de l\'application.</string>
    <!-- Subtitle displayed below the user who named the app -->
    <string name="special_thanks_name">Pour avoir donné le nom à l\'application.</string>
```

Reply:
Completely forgot how common ' is in French